src/lavinmq/replication/replicator.cr

```diff
@@ -0,0 +1,14 @@
+require "../mfile"
+
+module LavinMQ
+  module Replication
+    module Replicator
```

Comment:
Why not an `abstract class`?

Reply:
I like the idea of being able to use multiple interfaces when applicable to reduce the interface you need to mock/stub in specs. `Replication::Server` now implements `Replication::Replicator` and `Replication::FileIndex`.